### DISCUSSION TITLE
希望增加批量修改文章内图片地址的功能或插件

### DISCUSSION THREAD

**Discussion by dick86114** [halo-dev/halo#6510] [Q&A]
### 你当前使用的版本

2.18

### 描述一下此特性

因为是从wp迁移过来的数据，文章里大量的图片地址都还是wp的链接地址。
图片已经做了库的迁移，文章里只需要修改域名端口即可，但无法批量修改。

请问有什么办法可以批量修改所有文章里图片地址，或者哪怕是一篇文章里所有图片地址的办法么？

目前只能通过单个图片修改链接的方法进行手动修改，太繁琐了

### 附加信息

![image](https://github.com/user-attachments/assets/7d682921-4c96-4bd9-9acf-1af8b747dfe5)

**Comment by ruibaby** [comment]
解决图片的链接问题，最好的方式应该是代理，可以参考：https://halo-plugin-migrate.pages.dev/provider/wordpress.html#%E9%99%84%E4%BB%B6%E8%BF%81%E7%A7%BB

**Reply by dick86114** [reply]
这个数据我已经迁移，但我真正想做的是换域名，因此文章里的图片地址希望做批量替换，不知道大佬有没其他方案可以实现呢

**Comment by JohnNiang** [comment]
See https://github.com/orgs/halo-dev/discussions/6465 for more.
